paper/scope.org

```diff
@@ -0,0 +1,353 @@
+#+AUTHOR: Jeff Garland
+#+TITLE: D3610R0 Scope Guard for C++29
+#+EMAIL: jeff@crystalclearsoftware.com
+#+REVISION: 0
+#+SETUPFILE: https://fniessen.github.io/org-html-themes/org/theme-readtheorg.setup
+#+options: num:nil
+#+LATEX_CLASS: article
+#+LATEX_CLASS_OPTIONS: [11pt]
+#+LaTeX_HEADER: \usepackage[letterpaper,margin=.75in,top=.75in,bottom=.75in]{geometry}
+#+LaTeX_HEADER: \usepackage[frame=lines, linenos=true]{minted}
+#+LaTeX_HEADER: \usepackage{pdfpages}
+
+*Authors:*
+    Jeff Garland
+    Robert Andrzejuk
```

Comment:
@robert-andrzejuk I assume you want to be a co-author?

Reply:
Only if my contribution warrants it.

Reply:
hopefully so